rapid_response_xblock/static/html/rapid.html

```diff
@@ -1,6 +1,8 @@
 <div class="rapid-response-block" data-staff="{{ is_staff }}" data-open="{{ is_open }}">
   <div id="rapid-response-content">
   </div>
+  <div id="rapid-response-data">
```

Comment:
Nit: `rapid-response-results` is probably clearer

Reply:
This is actually unused in this PR so I'm removing the div but I'll make the change in the next one